src/main/java/me/zeroeightsix/kami/command/commands/VanishCommand.java

```diff
@@ -0,0 +1,38 @@
+package me.zeroeightsix.kami.command.commands;
+
+import me.zeroeightsix.kami.command.Command;
+import net.minecraft.client.Minecraft;
+import net.minecraft.entity.Entity;
+
+/**
+ * Created by d1gress/Qther on 25/11/2017.
+ */
+
+public class VanishCommand extends Command {
+
+    private static Entity vehicle;
+    Minecraft mc = Minecraft.getMinecraft();

```

Comment:
Why is this needed?

Reply:
wdym

Reply:
The mc =

You don't have to declare it?

Reply:
easier for me to port cos bleachhack predefines it in Command or something along the lines of that

Reply:
Nevermind, you're right yea

Reply:
its very not needed so feel free to change it